widget/playground/src/utils/colors.ts

```diff
@@ -10,15 +12,58 @@ export const getMainColor = (
   const isDarkTab = tab === 'dark';
   const isLightTab = tab === 'light';
   const isAutoTab = tab === 'auto';
+
   if (!!theme?.colors) {
-    if (isAutoTab && !theme.singleTheme) {
-      return (theme?.colors[mode || 'light'] || {})[key];
+    if (isAutoTab) {
```

Comment:
As we talked in tg, you can revert this change and only disable `Custom Color` when the selected preset isn't in the current tab.

Reply:
Done